internal/component/otelcol/exporter/faro/faro.go

```diff
@@ -16,12 +17,12 @@ import (
 func init() {
 	component.Register(component.Registration{
 		Name:      "otelcol.exporter.faro",
-		Community: true,
+		Stability: featuregate.StabilityGenerallyAvailable,
```

Comment:
```suggestion
		Stability: featuregate.StabilityExperimental,
```

Shouldn't this be experimental? The [upstream component](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/faroexporter/README.md) is still alpha.

Reply:
Sure, let's make it experimental. What is the criteria to change the stability level to Generally Available? Should the component be in Beta or Stable in contrib repo for that?